src/plot.jl

```diff
@@ -0,0 +1,31 @@
+using RecipesBase
+
+@recipe function f(ga::GeoArray)
+    xflip --> false
+    yflip --> true
+    aspect_ratio --> 1
+    seriestype := :heatmap
+    color := :deep
+
+    coords = centercoords(ga)
+    x = map(x->x[1], coords[:, 1])
+    y = map(x->x[2], coords[end, :])
+
+    xlims --> (minimum(x), maximum(x))
+    ylims --> (minimum(y), maximum(y))
+
+    z = ga[:,:,1]'  # only first layer for now
+
+    # Slice data and replace missing by nodata
+    dtype = eltype(ga)
+    if isa(dtype, Union) && dtype.a == Missing
+        dtype = dtype.b
+        nodata = typemax(dtype)
```

Comment:
Perhaps in the case of Float we can use NaN? Are you sure Plots.jl is not yet updated to show `Union{X, Missing}` properly without this block?